tests/verify-pool-stratis.yml

```diff
@@ -0,0 +1,54 @@
+---
+# Only when pool is stratis
+- name: Check Stratis options
+  when: storage_test_pool.type == 'stratis'
+  block:
+    - name: Ensure python3-jmespath is present (needed for json_query filter)
```

Comment:
It's best to avoid using json_query and use `selectattr` and `map` instead. See below.

Reply:
Reworked the check without `json_query`.